.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Yes, IntelliJ seems to have updated the project files handling since last time. So we do not have to add all config files. I only added the code-style. This way we can maintain it an people, get the update without reloading the code-style.